crates/scroll/chainspec/src/lib.rs

```diff
@@ -54,6 +56,8 @@ pub use scroll_sepolia::SCROLL_SEPOLIA;
 pub struct ScrollChainSpecBuilder {
     /// [`ChainSpecBuilder`]
     inner: ChainSpecBuilder,
+    /// [`ScrollChainConfig`]
+    config: ScrollChainConfig,
```

Comment:
this is added in order to have access to the `ScrollChainConfig` in the `ScrollExecutionStrategy`. Because of the current [API](https://github.com/paradigmxyz/reth/blob/735eb4b97cf3937886a2972647e0afb90c916a71/crates/cli/commands/src/stage/mod.rs#L44-L48) on the `Stage` execute method, this was the easiest way to add it.